app/seksjon/oppsummering/OppsummeringView.tsx

```diff
@@ -0,0 +1,265 @@
+import { Alert, BodyLong, VStack } from "@navikt/ds-react";
+import DinSituasjonOppsummeringV1 from "~/seksjon/din-situasjon/v1/oppsummering/DinSituasjonOppsummeringV1";
+import { useLoaderData } from "react-router";
+import { loader } from "~/routes/$soknadId.oppsummering";
+import BostedslandOppsummeringV1 from "~/seksjon/bostedsland/v1/oppsummering/BostedslandOppsummeringV1";
+import BarnetilleggOppsummeringV1 from "~/seksjon/barnetillegg/v1/oppsummering/BarnetilleggOppsummeringV1";
+import VernepliktOppsummeringV1 from "~/seksjon/verneplikt/v1/oppsummering/VernepliktOppsummeringV1";
+import UtdanningOppsummeringV1 from "~/seksjon/utdanning/v1/oppsummering/UtdanningOppsummeringV1";
+import ReellArbeidssøkerOppsummeringV1 from "~/seksjon/reell-arbeidssøker/v1/oppsummering/ReellArbeidssøkerOppsummeringV1";
+import TilleggOpplysningerOppsummeringV1 from "~/seksjon/tilleggsopplysninger/v1/oppsummering/TilleggOpplysningerOppsummeringV1";
+import PersonaliaOppsummering from "~/seksjon/personalia/v1/oppsummering/PersonaliaOppsummering";
+import EgenNæringOppsummeringV1 from "~/seksjon/egen-næring/v1/oppsummering/EgenNæringOppsummeringV1";
+import AnnenPengestøtteOppsummeringV1 from "~/seksjon/annen-pengestøtte/v1/oppsummering/AnnenPengestøtteOppsummeringV1";
+import { stegerISøknaden } from "~/routes/$soknadId";
+import ArbeidsforholdOppsummeringV1 from "~/seksjon/arbeidsforhold/oppsummering/ArbeidsforholdOppsummeringV1";
+import { JSX } from "react";
+
+export default function OppsummeringView() {
+  const loaderData = useLoaderData<typeof loader>();
+  if (!loaderData) {
+    return <></>;
+  }
+
+  function renderOppsummeringsSeksjon(
+    seksjonsId: string,
+    seksjonsUrl: string,
+    seksjonsData: string
+  ) {
+    if (!seksjonsData) return <></>;
+    const seksjonSvarene = JSON.parse(seksjonsData);
+
+    function printVersjonErrorMelding(seksjonsId: string, versjon: number) {
+      console.error(
+        `Ukjent versjon for seksjon ${seksjonsId}. Mottatt versjon: ${versjon}. 
+        Prøver å vise nyeste versjon av oppsummeringsmodulen for seksjonen.`
+      );
+    }
+
+    switch (seksjonsId) {
```

Comment:
Lang switch-case, men super enkelt å forstå. I like it!

Reply:
Man kunne vurdert å skille ut hver seksjon i en egen fil/komponent som håndterer sjekk av versjon. Men jeg er enig i at det er enkelt å forstå nå, så ikke noe krise!